index.js

```diff
@@ -54,19 +54,62 @@ module.exports = {
       let checker = new VersionChecker(this.parent).for('ember-cli-babel', 'npm');
 
       if (checker.satisfies('>= 6.0.0')) {
+
+        // Go find specified addons in the addons defined at the app level
+        let firstPassAddonObj = this._getConfig().coveredAddons.reduce((acc, addonName) => {
+            let addon = this.project.findAddonByName(addonName);
+            if (addon != null) {
+              acc.foundAddons[addonName] = addon;
+            } else {
+              acc.unfoundAddons.push(addonName);
+            }
+            return acc;
+          }, { foundAddons: {}, unfoundAddons: [] });
+        let finalAddonObj = firstPassAddonObj;
+
+        // Now go through each of the addons we found at the app level
+        // and see if our specified covered addons are sub dependencies
+        const MAX_LEVELS = 2;
```

Comment:
limiting how deep to go in the dependency tree to find addons.  Currently we stop after 2 passes (addon deps and addon's addon deps)